src/main/java/com/textmessenger/service/PostServiceImpl.java

```diff
@@ -80,6 +81,15 @@ public void deletePost(Post post) {
     return PostToFront.convertListPostsToResponse(postRepository.findAll(orderBy()));
   }
 
+  @Override
+  public List<PostToFront> getPage(int page, int size) {
+    List<Post> all = postRepository.findAll(orderBy());
+    return PostToFront.convertListPostsToResponse(all.stream()
+            .skip((int)page * size)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast to "int". ([squid:S1905](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1905))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=169&issues=AWWmXD78fnKVigJDaecG&open=AWWmXD78fnKVigJDaecG)

[ISSUE_KEY]: AWWmXD78fnKVigJDaecG